src/app/features/tech-record/components/tech-record-history/tech-record-history.component.html

```diff
@@ -19,12 +19,7 @@
       <td class="govuk-table__cell">{{ techRecord.createdAt | date: 'dd/MM/yyyy HH:mm' }}</td>
       <td class="govuk-table__cell">
         <ng-container *ngIf="techRecord.createdAt !== currentRecord?.createdAt">
-          <a
-            id="view-test-{{ vehicleTechRecord.vin }}"
-            href="/tech-records/{{ vehicleTechRecord.systemNumber }}/historic/{{ convertToUnix(techRecord.createdAt) }}"
-          >
-            View
-          </a>
+          <a id="view-test-{{ vehicleTechRecord.vin }}"> View </a>
```

Comment:
`href`/`routerlink` not needed anymore?

Reply:
Hmm that's interesting, the linter removed it...